contracts/utils/Swapper.sol

```diff
@@ -0,0 +1,66 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
Can we move this into a `test` directory instead of a `utils` directory for now since it's only used in tests?